streaming_jupyter_integrations/magics.py

```diff
@@ -369,6 +359,25 @@ def __enrich_cell(self, cell: str) -> str:
         joined_cell = inline_sql_in_cell(enriched_cell)
         return joined_cell
 
+    def __flink_execute_sql_file(self, path: str) -> None:
+        if self.background_execution_in_progress:
+            self.__retract_user_as_something_is_executing_in_background()
+            return
+
+        if os.path.exists(path):
+            with open(path, "r") as f:
+                statements = map(lambda s: self.__enrich_cell(s.rstrip(';')), sqlparse.split(f.read()))
+        else:
+            print("File {} not found".format(path))
```

Comment:
Where does it get printed? And are we sure we want to print bare `File init.sql not found` to the user whenever they initialize their magics? I think I would like to either not show this message at all if it is about `init.sql` or show with an additional information (like "Place init.sql in the same directory as the notebook if you would like to blablah" or something).